src/main/kotlin/com/terraformation/backend/accelerator/db/ApplicationStore.kt

```diff
@@ -215,6 +215,7 @@ class ApplicationStore(
 
     return dslContext
         .select(
+            APPLICATIONS.INTERNAL_NAME,
```

Comment:
Out of scope for this PR, but I don't love that we now have _three_ sources of truth for this value. We should figure out how to keep this in just one place.